content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -84,8 +84,8 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```bash
    chmod +x kubectl
    mkdir -p ~/.local/bin/kubectl
```

Comment:
A simpler fix would be change this to `mkdir -p ~/.local/bin`.

There is no need to create a subdirectory under `~/.local/bin`.

Reply:
yes , simple and short.